dotcom-rendering/src/components/Carousel.importable.tsx

```diff
@@ -413,17 +413,23 @@ const Title = ({
 	titleColour,
 	titleHighlightColour,
 	isCuratedContent,
+	url,
 }: {
 	title: string;
 	titleColour: string;
 	titleHighlightColour: string;
 	isCuratedContent?: boolean;
+	url?: string;
 }) =>
-	title === 'Videos' ? (
+	url ? (
 		<a
 			css={[linkStyles]}
-			href="https://www.theguardian.com/video"
-			data-link-name="video-container-title Videos"
+			href={url}
+			data-link-name={
+				title === 'Videos'
+					? 'video-container-title Videos'
```

Comment:
This specific data-link-name is a hangover from Frontend. I think we should use the more generic `section heading` for all carousels. However, I am waiting for sign off from D&I for this so I've kept this hardcoded here in the meantime so that this PR doesn't get blocked.